src/odemis/acq/fastem.py

```diff
@@ -155,6 +154,12 @@ def estimate_acquisition_time(self):
         return tot_time
 
     def _calculate_field_indices(self):
+        # TODO: Old method is used since polygonal fields is not yet supported by the gui. This must be adjusted when
+        #  it is.
+        indices = self.get_square_field_indices(self.coordinates.value)
+        return indices
+
+    def get_square_field_indices(self, coordinates):
```

Comment:
`:param coordinates:` left to be stated in docstring

Reply:
don't forget this comment